integr8ly/installation
# update launcher catalog release

## Additional Information
Updated catalog release to pull in the latest launcher catalog
## Verification Steps

- deploy this branch
- check you can deploy the launcher aggregation booster and that the build does not fail
- check the launcher.backend.catalog.git.ref is set to integreatly-1.2.2
